packages/instantsearch.js/src/types/index.ts

```diff
@@ -1,10 +1,4 @@
-// internal
-export * from './utils';
-
-// Algolia-related
-// eslint-disable-next-line import/export
-export * from './algoliasearch';
-export * from './results';
+export * from 'instantsearch-core/src/types';
```

Comment:
this file isn't exported, you can add it to `exports` of its package.json

Reply:
And just straight up have it as `instantsearch-core/types` too